test/checkpoint/recovery-test.sh

```diff
@@ -2,12 +2,26 @@
 
```

Comment:
Can we turn this script into an Evergreen function?

Reply:
@lukech  This script cannot be converted into an evergreen function, because this script gets called from an external script `run_parallel.sh`.